hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/UserGroupInformation.java

```diff
@@ -528,6 +528,10 @@ private HadoopLoginContext getLogin() {
   private void setLogin(LoginContext login) {
     user.setLogin(login);
   }
+  
+  private void setLastLogin(long now) {
```

Comment:
add javadoc, note unit of time. Milliseconds, right?

Reply:
Make `@param time` and `long now` the same variable name? e.g. name it `loginTime`

Reply:
Yes agreed, made the change